.github/workflows/ci.yaml

```diff
@@ -0,0 +1,12 @@
+name: Rust CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  build-and-test:
+    uses: iExecBlockchainComputing/github-actions-workflows/.github/workflows/rust-build.yml@rust-build-v2.0.0
+    with:
+      rust-version: '1.85.0'
+      enable-cache: true
```

Comment:
To avoid issues, and as there are not many properties, I would recommend to write all properties, even if they do not override default values.